gnucash/report/standard-reports/transaction.scm

```diff
@@ -1018,19 +1018,7 @@ Credit Card, and Income accounts."))))))
 
 
     (define (do-rows-with-subtotals splits
-                                    table
```

Comment:
Can this commit work without the next one ? If not (and it doesn't look like that to me), it shouldn't be two commits. Perhaps you forgot to squash them together ?

Reply:
I think you're right. I'll recheck.